.github/workflows/generate-changelog/action.yml

```diff
@@ -0,0 +1,25 @@
+name: Generate Changelog
+description: Generate a changelog for the current version with the last version as the base.
+
+inputs:
+  TOKEN:
+    required: true
+  REPOSITORY:
+    required: true
+  HEAD_BRANCH_NAME:
+    required: true
+  BASE_TAG_NAME:
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - run: |
+            npm install --no-package-lock --no-save @octokit/core@3.4.0
+            node ./.github/scripts/generate-changelog.js
+      shell: bash
```

Comment:
Move before `run`

Reply:
If the "shell: bash" will be before the "run" the action will not work.
As the docs also mentioned, I'm getting also warning on my IDE.

https://docs.github.com/en/actions/creating-actions/creating-a-composite-action